styles.css

```diff
@@ -1,3 +1,237 @@
 body {
+  background-image: url("./images/space.jpg");
+  background-size: cover;
+}
+
+.centerDiv{
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  margin: 30px
+}
+
+.name{
+  font-size: 32px;
+  font-weight: bold;
+  margin-top: 10px;
+  margin-bottom: 0px;
+}
+
+#main-display{
+  background-image: url("./images/techBG.png");
+  background-size: cover;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  border: 3px;
+  border-color: blue;
+  width: 70vw;
+  height: 55vh;
```

Comment:
If you ever face display problems, it is often related to vw and vh. Nothing wrong here, just a heads up